src/config/config.ts

```diff
@@ -298,7 +301,8 @@ export class Config implements IConfig {
   }
 
   get commandIDs() {
-    return uniq(this.commands.map(c => c.id))
+    const ids = Lodash.flattenDeep(this.commands.map(c => [c.id, c.aliases]))
```

Comment:
@amphro let me know your thoughts, but here I'm saying `commandIDs` should return alias ids as well

Reply:
maybe some downstream affects I'm not thinking through